### PR TITLE
docs/environment-setup.md added

### PR DESCRIPTION
**Pull request by tpmai22**
I have added environment setup docs for new dev which included troubleshooting guide to run the project on window

### CODE REVIEW

File: docs/environment-setup.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+To run the project on your local machine
+
+```bash
+pnpm run start
+```
+
+<hr>
+
+## Troubleshooting for starting the project on a Window machine:
```
Comment (DukeManh) [inline docs/environment-setup.md:52]: We should encourage people to install WSL2  so they don't run into problems with Windows CMD in the future

**Review by DukeManh — CHANGES_REQUESTED** [review]

File: docs/environment-setup.md (hunk 1 above)
Comment (tpmai22) [reply docs/environment-setup.md:52]: @DukeManh so I suggest to change to <h2> Encouraging to install WSL2 </h2>

File: docs/environment-setup.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+<hr>
+
+## Prerequisites:
+
+- [Node.js](https://nodejs.org/en/download/)
+- [Npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+### Install pnpm
+
+We use `pnpm` because ton of benefit that it brought to the table ([More on the benefits of pnpm](https://pnpm.io/motivation)).
```
Comment (DukeManh) [inline docs/environment-setup.md:23]:
```suggestion
We use `pnpm` as our package manager. 
Learn more about [pnpm](https://pnpm.io/motivation)
```

File: docs/environment-setup.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+### Table of Contents
+
+[Prerequisites:](#prerequisites)
+
+- [Install pnpm](#install-pnpm)
+
+[After installing the prerequisites and pnpm:](#after-installing-the-prerequisites-and-pnpm)
+
+- [Instal project dependencies](#install-project-dependencies)
+- [Install WSL2 on Window](#install-wsl2-on-window)
```
Comment (DukeManh) [inline docs/environment-setup.md:12]:
```suggestion
- [Install WSL2 on Window](#install-wsl2-on-window)
- [Instal project dependencies](#install-project-dependencies)
```

File: docs/environment-setup.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+pnpm install
+```
+
+To run the project on your local machine
+
+```bash
+pnpm run start
+```
+
+## Install WSL2 on Window
```
Comment (DukeManh) [inline docs/environment-setup.md:49]: Move this up before installing dependencies because it's the first step

**Review by DukeManh — CHANGES_REQUESTED** [review]

File: docs/environment-setup.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+# Environment Setup
+
+### Table of Contents
+
+[Prerequisites:](#prerequisites)
+
+- [Install pnpm](#install-pnpm)
+- - [Install WSL2 on Window](#install-wsl2-on-window)
```
Comment (DukeManh) [inline docs/environment-setup.md:8]: hmm,

**Review by DukeManh — CHANGES_REQUESTED** [review]

File: docs/environment-setup.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+[After installing the prerequisites and pnpm:](#after-installing-the-prerequisites-and-pnpm)
```
Comment (DukeManh) [inline docs/environment-setup.md:10]: Start the Frontend development server

**Review by DukeManh — CHANGES_REQUESTED** [review]

File: docs/environment-setup.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
+# Environment Setup
+
+### Table of Contents
```
Comment (DukeManh) [inline docs/environment-setup.md:3]: Could you make this `h2 - ##`, directly following `h1`.

File: docs/environment-setup.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+```bash
+npm install -g pnpm
+```
+
+<hr>
+
+## After installing the prerequisites and pnpm:
+
+## Install project dependencies
```
Comment (DukeManh) [inline docs/environment-setup.md:49]: This should be `###`

File: docs/environment-setup.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+- [Instal project dependencies](#install-project-dependencies)
+
+<hr>
+
+## Prerequisites:
+
+- [Node.js](https://nodejs.org/en/download/)
+- [Npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+## Install WSL2 on Window
```
Comment (DukeManh) [inline docs/environment-setup.md:20]: `###`

**Review by DukeManh — CHANGES_REQUESTED** [review]

**Review by DukeManh — APPROVED** [review]

**Comment by DukeManh** [comment]
Looks great, thank James. Can you merge it yourself?

**Pull request merged**